main.c

```diff
@@ -1374,9 +1379,17 @@ void loop() {
 }
 
 int main() {
+  XVideoSetMode(640, 480, 32, REFRESH_DEFAULT);
+	size_t fb_size = 640 * 480 * 4;
```

Comment:
Keep these lines separated to show that these are different things.

- Setting video mode is a must (there are currently no plans to remove it - even if we could, we might want to force 720p or something).
- Doing the fb allocation is a hack that's only required temporarily. I usually `#if 1` to imply that it is something that can be turned off / disabled in the future.